virtual_rainforest/models/litter/litter_pools.py

```diff
@@ -178,6 +286,111 @@ def calculate_litter_decay_structural_above(
     )
 
 
+def calculate_litter_decay_woody(
+    temperature_factor: NDArray[np.float32],
+    litter_pool_woody: NDArray[np.float32],
+    litter_decay_coefficient: float,
+) -> NDArray[np.float32]:
+    """Calculate decay of the woody litter pool.
+
+    This function is taken from :cite:t:`kirschbaum_modelling_2002`.
+
+    Args:
+        temperature_factor: A multiplicative factor capturing the impact of temperature
+            on litter decomposition [unitless]
+        litter_pool_woody: The size of the woody litter pool [kg C m^-2]
+        litter_decay_coefficient: The decay coefficient for the woody litter pool
+            [day^-1]
+
+    Returns:
+        Rate of decay of the woody litter pool [kg C m^-2 day^-1]
+    """
+
+    # Factor capturing the impact of litter chemistry on decomposition, calculated based
+    # on formula in Kirschbaum and Paul (2002) with the assumption that dead wood is 50%
+    # lignin. Keeping as a hard coded constant for now, as how litter chemistry is dealt
+    # with is going to be revised in the near future.
+    litter_chemistry_factor = 0.082085
+
+    return (
+        litter_decay_coefficient
+        * temperature_factor
+        * litter_pool_woody
+        * litter_chemistry_factor
+    )
+
+
+def calculate_litter_decay_metabolic_below(
+    temperature_factor: NDArray[np.float32],
+    moisture_factor: NDArray[np.float32],
+    litter_pool_below_metabolic: NDArray[np.float32],
+    litter_decay_coefficient: float,
+) -> NDArray[np.float32]:
+    """Calculate decay of below ground metabolic litter pool.
+
+    This function is taken from :cite:t:`kirschbaum_modelling_2002`.
+
+    Args:
+        temperature_factor: A multiplicative factor capturing the impact of temperature
+            on litter decomposition [unitless]
+        moisture_factor: A multiplicative factor capturing the impact of soil moisture
+            on litter decomposition [unitless]
+        litter_pool_below_metabolic: The size of the below ground metabolic litter pool
+            [kg C m^-2]
+        litter_decay_coefficient: The decay coefficient for the below ground metabolic
+            litter pool [day^-1]
+
+    Returns:
+        Rate of decay of the below ground metabolic litter pool [kg C m^-2 day^-1]
+    """
+
+    return (
+        litter_decay_coefficient
+        * temperature_factor
+        * moisture_factor
+        * litter_pool_below_metabolic
```

Comment:
Is the `litter_chemistry_factor` not relevant here? I don't know much about the science but you used it for the other two new pools

Reply:
Ahh right yes this could be clearer, the metabolic pools are for biological matter that doesn't contain recalcitrant polymers (i.e. lignin). The `litter_chemistry_factor` is a placeholder for actually tracking the recalcitrant polymers, so is not needed in either metabolic pool

Reply:
I'll add a description of the pools to the module docstring as there isn't all that much info in there at the moment